stress/v2/statement/go.go

```diff
@@ -0,0 +1,35 @@
+package statement
+
+import (
+	"fmt"
+
+	"github.com/influxdata/influxdb/stress/v2/ponyExpress"
+)
+
+// GoStatement is a Statement Implementation to allow other statements to be run concurrently
+type GoStatement struct {
+	Statement
+
+	StatementID string
+}
+
+// SetID statisfies the Statement Interface
+func (i *GoStatement) SetID(s string) {
+	i.StatementID = s
+}
+
+// Run statisfies the Statement Interface
+func (i *GoStatement) Run(s *ponyExpress.StoreFront) {
```

Comment:
What is this supposed to do? It doesn't seem intuitive to me, so there might be a simpler way to achieve it.